ext/crypto/encrypt.rs

```diff
@@ -136,3 +159,59 @@ fn encrypt_aes_cbc(
   };
   Ok(ciphertext)
 }
+
+fn encrypt_aes_gcm(
+  key: RawKeyData,
+  length: usize,
+  tag_length: usize,
+  iv: Vec<u8>,
+  additional_data: Option<Vec<u8>>,
+  data: &[u8],
+) -> Result<Vec<u8>, deno_core::anyhow::Error> {
```

Comment:
Is there a specific reason to use `deno_core::anyhow::Error` here instead of `AnyError`?

Reply:
Fixed. I'm not sure though It was being used for other `*_decrypt` functions too